### ISSUE TITLE
Write code for automatic arm moving

### ISSUE THREAD

**Issue by ddsnowboard** [RoboRebel/RoboRebels2016#4]
The feature in #1 was originally for moving the arm automatically to the shooting and picking-up position, but it morphed into safety code, leaving a vacuum where it used to be. Now we have to write some new code that does what that code used to do. 

We should re-assign the right and left bumpers to move directly to the shooting and picking-up positions. I don't know which one should be which, but if @ack6600 writes this, he is as good a person as any to make that decision because he might be driving. When you press one of those buttons, the arm will automatically go to the right position _without the driver having to hold the button_. Of course, as is practically dictated by the implementation, if the driver presses one of the buttons that manually move the arm, they will override the automatic motion. 

Finding the target angles could be difficult. Here are my two cents. The whole reason that we need this system is because it is really hard to find the angles manually, even in the pits, so we can't have another system like the one in #1 to set these numbers. My idea is that we use the values we set as safeties to find the target angles. The highest angle will probably be different every time we set it, and that's OK, but that also means that we can't use it to calculate anything. The lowest angle, though, will be on the floor, which is easy to find and should be pretty repeatable. Therefore, I suggest we figure out the optimum angles for shooting and sucking at school, figure out how far they are from the floor, and then calculate the angles in the code that way. 

In short:

``` Java
public static double shootAngle = potLowestAngle + SHOOT_ANGLE_OFFSET;
public static double suckAngle = potLowestAngle + SUCK_ANGLE_OFFSET;
```

Any thoughts?

**Comment by ddsnowboard** [comment]
This was kinda started in 01c58c9fd74d1d1a19ff99aceaa9947617d960dd, but I haven't been able to test since we don't have an arm. I hope that we don't have to roll a bunch of stuff back due to buggy code and not enough time to test it.
